demos/llm_demo/cpp/README.md

```diff
@@ -0,0 +1,41 @@
+# LLM C++ Demo
+
+This application showcases inference of s large language model (LLM). Unlike most of other demos, this application doesn't have a rich set of command line arguments to encourage the reader to explore and modify the source code.
+
+## How it works
+
+The demo loads a model (`.xml` and `.bin`) to OpenVINO™ and a provided vocab (`.gguf`) to use for tokinezation. The model is reshaped to batch 1 and variable prompt size. A prompt is tokenized and passed to the model. The model greedily generates token by token until the special end of sentence token is obtained. The predicted tokens are converted to chars and printed in a streaming fashion.
+
+## Supported models
+
+### LLaMA 2
+
+Follow https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/254-llm-chatbot to convert the model to intermediate representation (IR) (`.xml` and `.bin`) format. Use `<omz_dir>/demos/thirdparty/llama.cpp/models/ggml-vocab-llama.gguf` as a vocab.
+
+### OpenLLaMA
+
+1. Install dependencies
+
+   ```sh
+   git lfs install
+   python -m pip install --extra-index-url https://download.pytorch.org/whl/cpu onnx git+https://github.com/huggingface/optimum-intel.git
+   ```
+
+2. Download and convert the model
+
+   ```sh
+   git clone https://huggingface.co/openlm-research/open_llama_3b_v2
```

Comment:
Would it be easier to use optimum-cli instead? we recommend this now I think.

Reply:
Moved